public/scripts/auth.js

```diff
@@ -87,6 +89,19 @@ friendlyPix.Auth = class {
       }
     });
   }
+
+  deleteAccount() {
+    this.auth.currentUser.delete().then(function() {
```

Comment:
You can use ES2015 style arrow functions since we are transpiling the code (here and in the catch below use `=> {`)